src/analysis/if_statements.d

```diff
@@ -117,6 +117,17 @@ private struct ExpressionInfo
 			return 0;
 		return -1;
 	}
+	bool opEquals(ref const ExpressionInfo other) const nothrow
```

Comment:
Actually, are these functions added because of `object_const_check`? If that's the case I think this is a false positive, since the compiler already generates those functions.